Library/Homebrew/cmd/tests.rb

```diff
@@ -1,9 +1,34 @@
 require "fileutils"
+require "official_taps"
 
 module Homebrew
   def tests
+    ENV["HOMEBREW_NO_ANALYTICS_THIS_RUN"] = "1"
+    Homebrew.install_gem_setup_path! "bundler"
+
+    if ARGV.include? "--official-cmd-taps"
```

Comment:
Quickly getting into bike-shedding territory, but maybe `--official-external-commands`? Sounds a bit clearer to me.